src/lib/level_manager.py

```diff
@@ -1,7 +1,11 @@
 from src.game_levels import development_playground
```

Comment:
this file is where i wonder if my integration is optimal, hopefully not too shoehorned in